docs/source/using/managing-minishift.adoc

```diff
@@ -85,9 +85,17 @@ $ minishift start --iso-url https://github.com/minishift/minishift-centos-iso/re
 ----
 
 - Manually download the {project} CentSO ISO image from the link:https://github.com/minishift/minishift-centos-iso/releases[releases page] and enter the file URI to the image:
-+
+
+[NOTE]
+====
+The path given for a file URI must be an absolute path.
+
+On GNU/Linux or macOS, the path must begin with *_/_*.
+
+On Windows, the path must begin with a drive letter (*_C:_*, *_D:_*).
+====
 ----
-$ minishift start --iso-url file:///<path_to_ISO_image>
+$ minishift start --iso-url file://<absolute_path_to_ISO_image>
```

Comment:
This is wrong I think (at least on macOS) as the two slashes `//` does not work. However the 3 slashes work fine.
```
$ minishift start --iso-url file://Users/lmohanty/Downloads/minishift-iso/minishift-b2d.iso
-- Checking if xhyve driver is installed ...
   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
   Checking for setuid bit ... OK
-- Checking the ISO URL ... FAIL
   See the 'Managing Minishift' topic for more information
```
```
$ minishift start --iso-url file:///Users/lmohanty/Downloads/minishift-iso/minishift-b2d.iso
-- Checking if xhyve driver is installed ...
   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
   Checking for setuid bit ... OK
-- Starting local OpenShift cluster using 'xhyve' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB
-- Starting Minishift VM ...................... OK
-- Checking for IP address ... OK
-- Checking if external host is reachable from the Minishift VM ...
   Pinging 8.8.8.8 ... OK
-- Checking HTTP connectivity from the VM ...
   Retrieving http://minishift.io/index.html ... OK
-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 0% used OK
-- OpenShift cluster will be configured with ...
   Version: v3.6.0
-- Checking `oc` support for startup flags ...
   host-config-dir ... OK
   host-data-dir ... OK
   host-pv-dir ... OK
   host-volumes-dir ... OK
   routing-suffix ... OK
```

Reply:
@LalatenduMohanty an absolute path is from `/`... which means `file://` + `/my/absolute/path` becomes `file:///my/absolute/path`.

Maybe an example will highlight this fact better.

Reply:
@gbraad Right, but adding two examples would be noisy, I think more elaborate examples, for both OS can go into the FAQ (if this gets asked a lot). and only an example absolute path for *nix systems in the docs.

Reply:
@anjannath I don't think two examples would be excessively noisy. Having more detailed information about using the `file://` scheme with absolute paths in the FAQ certainly makes sense to me, though.

As it is, the absolute path itself is explained clearly, but we don't explain how this interacts with `file://` -- this leads directly to the confusion regarding two versus three slashes. The two slashes are part of the `file://` scheme, whereas the third slash is the beginning of the absolute path on Linux/macOS.

By the way, if we only have an example of `--iso-url` using a `file://` scheme on Linux/macOS, then we'll encounter confusion from Windows users who are using three slashes because that's what was demonstrated in the documentation (`file:///c:/path/to/iso` and `file://c:/path/to/iso` are not equivalent).

Having an example `file://` URL for both Linux/macOS and Windows makes sense to me given the potential for confusion.

If we can explain this in a clear and concise way, I think that would be the best solution.

@gbraad @LalatenduMohanty Thoughts?

Reply:
> Having an example file:// URL for both Linux/macOS and Windows makes sense to me given the potential for confusion.

@robin-owen Agree. I would suggest to go for two example i..e. one for Windows and one for Linux/macOS. 

In general the idea is to give the information in a way which would be easier to read and understand.

Reply:
```
NOTE: The path given for a file URI must be an absolute path.

- On GNU/Linux or macOS, the path must begin with *_/_*. For example:
+
----
$ minishift start --iso-url file:///path/to/image.iso
----

- On Windows, the path must begin with a drive letter (*_C:_*, *_D:_*). For example:
+
----
$ minishift start --iso-url file://c:/path/to/image.iso
----
```

I think now it is clear enough, with both the examples. @robin-owen @LalatenduMohanty what do you guys think?

Reply:
That's absolutely fine with me, I'd just continue using a block for the admonition so that we can keep all of the information inside of the note more clearly/easily:

```adoc
[NOTE]
====
// info here
====
```

As written in your comment, only the sentence "The path given for a file URI must be an absolute path" would be placed in the note. :)